frontend/src/component/feature/Dependencies/AddDependencyDialogue.tsx

```diff
@@ -110,16 +117,34 @@ const useManageDependency = (
     const manageDependency = async () => {
         try {
             if (isChangeRequestConfiguredInAnyEnv()) {
-                await handleAddChange(
+                const actionType =
                     parent === REMOVE_DEPENDENCY_OPTION.key
                         ? 'deleteDependency'
-                        : 'addDependency',
-                );
+                        : 'addDependency';
+                await handleAddChange(actionType);
+                trackEvent('dependent_features', {
+                    props: {
+                        eventType:
+                            actionType === 'addDependency'
+                                ? 'add dependency added to change request'
+                                : 'delete dependency added to change request',
```

Comment:
Very minor thing, but these two are a little confusing at first. Maybe we can rephrase them a bit?

Something like "dependency added in change request" and "dependency removed in change request"? Or "dependency added (change request)"/"dependency removed (change request)". Or we could use multiple props? Something like this:

```js
eventType: 'dependency added',
where: 'change request'
```

Primarily to avoid the "add x added" or "delete x added". But eh, as long as we understand what we're talking about I think it's fine 😄

Reply:
I was thinking about it myself. Originally I wanted to say "dependency added in change requests" but it's actually not true. We're only adding "add dependency" change to a change request. We'll know that dependency was added in change request once we apply the change request which may never happen.